a8cteam51/build-processes-demo
# Add `wp i18n` command to generate POT files

#### Changes proposed in this Pull Request

* This PR adds the `i18n` command from [WP-CLI](https://github.com/wp-cli/i18n-command) to generate POT files.

#### Testing instructions



* Run `composer update`.
* There's a composer script `makepot` that should help as an alias to the `i18n` command. To build POT for the theme `build-processes-demo` we could run it like so:
  * `composer makepot:theme`.
  * Note that there's no `__` function anywhere in the theme, so you must first add one for it to be included in the POT file.
* We can also generate POT files for the `mu-plugin/build-processes-demo-blocks`:
  * `composer makepot:blocks`.

#### Notes

Since I was working on adding `phpcs` as a composer dependency as well I thought that we might use `wp i18n` for POT generation instead of a npm package. We can even remove the composer dependency if we assume everyone has WP-CLI installed.
